modules/ROOT/pages/6.0-release-notes-core-changes.adoc

```diff
@@ -0,0 +1,296 @@
+= TinyMCE 6.0: the core changes
+:navtitle: Core changes
+:description: The core changes for TinyMCE 6.0
+:keywords: releasenotes, bugfixes, defaults
+
+This section documents the three core changes made to {productname} 6.0.
+
+This section also includes four tables that set out the:
+
+* renamed APIs, Commands, Plugins, Options, and UI &amp; UX elements
+* values changed in a default {productname} 6.0 instance.
+* previously deprecated and now removed items.
+* never documented, and consequently never supported, items that have now been removed.
+
+
+* xref:core-changes[Core changes]
+* xref:things-we-renamed[Things we renamed]
+* xref:default-value-changes[Default value changes]
+* xref:previously-deprecated-items-now-removed[Previously deprecated items now removed]
+* xref:previously-undocumented-items-removed-without-prior-deprecation[Previously undocumented items removed without prior deprecation]
+
+
+// tag::core-changes[]
+[[core-changes]]
+== Core changes
+
+. {productname} 6.0 now requires a browser capable of running https://tc39.es/ecma262/2020/[ECMAScript 2020] (ES2020).
```

Comment:
We probably don't want this specifically advertised... our support is the last 2 major versions of supported browsers and the latest LTS release (e.g. Firefox ESR). That's as much as we support, so realistically it's actually higher than ES2020.

See the note at the bottom of https://www.tiny.cloud/docs/general-configuration-guide/system-requirements/#supportedwebbrowsers.